index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="calcyoulater.css">
+    <script src="calcyoulater.js"></script>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Viewport? Someone's done some HTML/CSS before 😄

Reply:
Well yes, but in this case it's more that I'm learning the joys of Emmet shortcodes in VS Code. Behold the glory of the !